tools/recording/parameter_generator.py

```diff
@@ -75,10 +75,13 @@ def generate_sample(self):
                     return round_sample
 
             elif self._ops == "linear":
-                return [
-                    self.round(sample[0][0]),  # in_features
-                    self.round(sample[1][0]),  # out_features
-                ]
+                in_features = self.round(sample[0][0])
+                out_features = self.round(sample[1][0])
+                if in_features != 0 and out_features != 0:
+                    return [
+                        self.round(sample[0][0]),  # in_features
```

Comment:
Why not return `in_features` and `out_features` here?

Reply:
changed to in_features and out_features